service/app/controller/mapper/RequestMapper.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * 
+ */
+package controller.mapper;
+
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.responsecode.ResponseCode;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import play.libs.Json;
+
+/**
+ * This class will map the requested JSON data into custom class.
+ * if request data is incorrect format then it will throw ProjectCommonException 
+ * with 400 error code.
+ * @author Manzarul
+ *
+ */
+public class RequestMapper {

```

Comment:
Have this reusable class in a common-service project which can be added as dependency by any sunbird micro-service.

Reply:
Class is deleted